definitions/divorce/json/CaseEventToFields/CaseEventToFields-payment-by-account.json

```diff
@@ -0,0 +1,198 @@
+[
```

Comment:
is this prod or nonprod file?

Have you updated unit tests to make sure all works?